spec/views/curation_concerns/base/_form_progress.html.erb_spec.rb

```diff
@@ -64,4 +65,16 @@
       end
     end
   end
+
+  context "when the work has been saved before" do
+    let(:work) do
+      gw = GenericWork.new(creator: ["somebody"], title: ["abc"])
```

Comment:
I tried using a `before { allow(work).to receive(:new_record?).and_return(false) }` like I did on the other test but that did not work. RSpec reported that it could not find a route when the id was null or something like that.